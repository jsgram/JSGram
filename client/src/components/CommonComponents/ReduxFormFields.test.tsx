import { renderField } from './ReduxFormFields';

import { shallow } from 'enzyme';
import React from 'react';

describe('renderField component:', () => {
    const props = {
        input: 'someintput',
        type: 'sometype',
        className: 'someclassName',
        placeholder: 'someplaceholder',
        meta: {
            error: 'somemetaerror',
            touched: true,
        },
    };

    let renderer: any;
    let Input: any;

    beforeEach(() => {
        Input = jest.fn(() => (<div></div>));
        // @ts-ignore
        renderer = shallow(<renderField {...props} />);
    });

    test('render - success', () => {
        expect(renderer).toMatchSnapshot();
    });
});
