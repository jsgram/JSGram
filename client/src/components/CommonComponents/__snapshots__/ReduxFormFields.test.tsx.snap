// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderField component: render - success 1`] = `ShallowWrapper {}`;
